charts/sauce-connect/templates/deployment.yaml

```diff
@@ -38,6 +38,13 @@ spec:
               value: /etc/config/sauce-connect.yaml
             - name: SAUCE_API_ADDRESS
               value: :{{ .Values.api.port }}
+            {{- if .Values.config.tunnel_pool }}
+            - name: SAUCE_TUNNEL_POOL
+              value: "true"
```

Comment:
This is not needed. If they put to config that's fine.

Reply:
it's there to avoid evaluating the pool size if it's set. Might not be the most elegant templating but it's logically correct